spec/controllers/requests_controller_spec.rb

```diff
@@ -0,0 +1,139 @@
+require "rails_helper"
+
+RSpec.describe RequestsController, type: :controller do
+  include SessionsHelper
+  let(:account_with_user){create(:account, :with_user, confirmed_at: Time.current)}
+  let!(:request_record) { create(:request, user: account_with_user.user) }
+  let!(:borrow_book) { create(:borrow_book, request: request_record) }
+
+  before do
+    sign_in account_with_user
+  end
+
+  describe "GET #new" do
```

Comment:
khi nào dùng describer, context, it

Reply:
- describe: Sử dụng khi để nhóm các test liên quan đến một đối tượng hoặc hành vi cụ thể.

> Ví dụ: Khối describe này nhóm tất cả các test liên quan đến action new của controller.

- context: Sử dụng khi để nhóm các test dựa trên các điều kiện hoặc tình huống khác nhau.

> Ví dụ: Ngữ cảnh khi người dùng có sách trong giỏ hàng.

- it: Sử dụng khi để định nghĩa các test case cụ thể, mỗi it block là một test case.

>  Ví dụ: kiểm tra xem khi gọi action new, biến instance @request có được khởi tạo là một đối tượng mới của Request hay không.